scripts/get_version.sh

```diff
@@ -4,4 +4,4 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-VERSION=$(yarn --silent json -f package.json version)
+VERSION=$(node -p "require('./package.json').version")
```

Comment:
pour etre plus explicit et lisible, je recommande toujours les options en versions longues dans le code versionné

```suggestion
VERSION=$(node --print "require('./package.json').version")
```